docs/pin.rst

```diff
@@ -199,11 +208,22 @@ its own to that.
         Return ``True`` if the pin is being touched with a finger, otherwise
         return ``False``.
 
+        .. note::
+            The default touch mode for the pins on the edge connector is 
+            `resistive`. The default for the logo pin **V2** is `capacitive`.
+
+        **Resitive touch**
         This test is done by measuring how much resistance there is between the
         pin and ground.  A low resistance gives a reading of ``True``.  To get
         a reliable reading using a finger you may need to touch the ground pin
         with another part of your body, for example your other hand.
 
+        **Capacitive touch**
+        This test is done by interacting with the electric field of a capacitor
+        using the a finger as a conductor. `Capacitive touch
```

Comment:
`the a finger` need to remove `the` or `a`

Reply:
updated in https://github.com/bbcmicrobit/micropython/pull/692/commits/78bd2418a5e2b22c538b233e3313806f2dc032e5